R/installspec.r

```diff
@@ -14,11 +20,11 @@ installspec <- function(user = TRUE) {
             break
         }
     }
-    
+
     if (!found_binary)
-        stop('Jupyter or IPython 3.0 has to be installed but could neither run “jupyter” nor “ipython”, “ipython2” or “ipython3”.
-             (Note that “ipython2” is just IPython for Python 2, but still may be IPython 3.0)')
-    
+        stop('Jupyter or IPython 3.0 has to be installed but could neither run "jupyter" nor "ipython", "ipython2" or "ipython3".
+             (Note that "ipython2" is just IPython for Python 2, but still may be IPython 3.0)')
```

Comment:
-1 to this. It's making the code uglier and more complex for no good reason. Programmers are used to seeing plain ascii quotes anyway.